import_export_batches/models.py

```diff
@@ -3209,83 +3198,50 @@ def import_candidate_positions_from_endorsement_json(self, batch_uri, batch_set_
+                        batch_header_column_007='position_description',
+                        batch_header_column_008='office_name',
+                        batch_header_column_009='state_code',
+                        batch_header_column_010='election_day',
+                        batch_header_column_011='google_civic_election_id',
                     )
                     batch_header_id = batch_header.id
 
                     if positive_value_exists(batch_header_id):
                         # Save an initial BatchHeaderMap
                         batch_header_map = BatchHeaderMap.objects.create(
                             batch_header_id=batch_header_id,
-                            batch_header_map_000='candidate_id',
-                            batch_header_map_001='candidate_name',
-                            batch_header_map_002='stance',
-                            batch_header_map_003='percent_rating',
-                            batch_header_map_004='grade_rating',
-                            batch_header_map_005='candidate_ocd_division_id',
-                            batch_header_map_006='organization_position_url',
-                            batch_header_map_007='organization_twitter_handle',
-                            batch_header_map_008='candidate_party_name',
-                            batch_header_map_009='candidate_twitter_handle',
-                            batch_header_map_010='facebook_url',
-                            batch_header_map_011='more_info_url',
-                            batch_header_map_012='profile_image_url',
-                            batch_header_map_013='profile_image_url_https',
-                            batch_header_map_014='statement_text',
-                            batch_header_map_015='candidate_office_name',
-                            batch_header_map_016='state_code',
-                            batch_header_map_017='office_ocd_division_id',
-                            batch_header_map_018='election_day',
-                            batch_header_map_019='google_civic_election_id',
+                            batch_header_map_000='candidate_name',
+                            batch_header_map_001='stance',
+                            batch_header_map_002='percent_rating',
+                            batch_header_map_003='grade_rating',
+                            batch_header_map_004='organization_twitter_handle',
+                            batch_header_map_005='candidate_twitter_handle',
+                            batch_header_map_006='more_info_url',
+                            batch_header_map_007='statement_text',
+                            batch_header_map_008='contest_office_name',
+                            batch_header_map_009='state_code',
+                            batch_header_map_010='election_day',
+                            batch_header_map_011='google_civic_election_id',
```

Comment:
For incoming positions, we need to support `organization_position_url` in addition to `more_info_url`